janus_core/calculations/single_point.py

```diff
@@ -228,6 +227,44 @@ def set_calculator(
         else:
             self.struct.calc = calculator
 
+    @property
+    def properties(self) -> Sequence[Properties]:
+        """
+        Physical properties to be calculated.
+
+        Returns
+        -------
+        Sequence[Properties]
+            Physical properties.
+        """
+        return self._properties
+
+    @properties.setter
+    def properties(self, value: MaybeSequence[Properties]) -> None:
+        """
+        Setter for `properties`.
+
+        Parameters
+        ----------
+        value : MaybeSequence[Properties]
+            Physical properties to be calculated.
+        """
+        self._properties = value
```

Comment:
Probably want the setting to be the last thing done? Not sure how this plays with certain failure modes.

Reply:
True, probably slightly safer, fixed